Examples/ArcGISToolkitExamples/Misc/CalibrationView.swift

```diff
@@ -0,0 +1,254 @@
+            elevationLabel.bottomAnchor.constraint(equalTo: headingLabel.topAnchor, constant: -24)
+        ])
+
+        addSubview(elevationSlider)
+        elevationSlider.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            elevationSlider.leadingAnchor.constraint(equalTo: elevationLabel.trailingAnchor, constant: 16),
+            elevationSlider.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor, constant: -16),
+            elevationSlider.centerYAnchor.constraint(equalTo: elevationLabel.centerYAnchor)
+        ])
+
+        // Setup actions for the two sliders. The sliders operate as "joysticks", where moving the slider thumb will start a timer
+        // which roates or elevates the current camera when the timer fires.  The elevation and heading delta
+        // values increase the further you move away from center.  Moving and holding the thumb a little bit from center
+        // will roate/elevate just a little bit, but get progressively more the further from center the thumb is moved.
+        headingSlider.addTarget(self, action: #selector(headingChanged(_:)), for: .valueChanged)
+        headingSlider.addTarget(self, action: #selector(touchUpHeading(_:)), for: [.touchUpInside, .touchUpOutside])
+
+        elevationSlider.addTarget(self, action: #selector(elevationChanged(_:)), for: .valueChanged)
+        elevationSlider.addTarget(self, action: #selector(touchUpElevation(_:)), for: [.touchUpInside, .touchUpOutside])
+
+        elevationSlider.isHidden = !elevationControlVisibility
+        elevationLabel.isHidden = !elevationControlVisibility
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
+        // If the user tapped in the view (and not in the sliders), do not handle the event.
+        // This allows the view below the calibration view to handle touch events.  In this case,
+        // that view is the SceneView.
+        let hitView = super.hitTest(point, with: event)
+        if hitView == self {
+            return nil
+        } else {
+            return hitView
+        }
+    }
+
+    // The timers for the "joystick" behavior.
+    private var elevationTimer: Timer?
+    private var headingTimer: Timer?
+    
+    /// Handle an elevation slider value-changed event.
+    ///
+    /// - Parameter sender: The slider tapped on.
+    @objc
+    func elevationChanged(_ sender: UISlider) {
+        if elevationTimer == nil {
+            // Create a timer which elevates the camera when fired.
+            elevationTimer = Timer(timeInterval: 0.25, repeats: true) { [weak self] (_) in
+                let delta = self?.joystickElevation() ?? 0.0
+//                print("elevate delta = \(delta)")
```

Comment:
Stands out a bit. I know it's just an example view, but should we check in commented out code?

Reply:
Yep. It's totally fine. It just stood out as I scrolled through :)